backend/models/Link.js

```diff
@@ -0,0 +1,38 @@
+module.exports = function(sequelize, Sequelize) {
+  let Link = sequelize.define(
+    "links",
+    {
+      id: {
+        type: Sequelize.INTEGER,
+        autoIncrement: true,
+        primaryKey: true
+      },
+      siteName: {
+        type: Sequelize.STRING
+      },
+
+      linkTitle: {
+        type: Sequelize.STRING
+      },
+      url: {
+        type: Sequelize.STRING
+      },
+      storyTags: {
+        type: Sequelize.STRING
+      },
+      timesClicked: {
+        type: Sequelize.INTEGER
+      }
```

Comment:
Need to add a `createdAt` field to fulfill LinkListView requirement of being able to sort by most recently added links.

Reply:
sequelize creates this for us